workflows/syncing-production-data-to-qa.md

```diff
@@ -0,0 +1,135 @@
+
+I. Day before the ILS sync:
+  1. Staff should be informed that writes to Sierra Test are discouraged / may not be reflected in RC
+  2. LSP Eng [disable Bib, Item, and Holding pollers in qa](#enabling-disabling-the-pollers)
+  3. Ops syncs Bib, Item, and Holding Service dbs from production > qa
+  4. (simultaneously to the Ops work above) LSP Eng copies Elastic production index over qa index
+II. Day of ILS sync:
+  1. Innovative kicks off the Sierra Prod > Test copy
+  2. When Innovative signals Sierra Test is ready for testing, LSP Eng confirms and reenables the Bib, Item, and Holdings pollers to pick up Sierra Test changes since yesterday
+  3. Within 1-2 hours, the Bib, Item, and Holding service dbs should be caught up and match the refreshed Sierra Test and associated discovery-api records also correctly indexed.
+
+## Enabling/Disabling the Pollers
+
+The pollers are controlled by EventBridge triggers:
+ - [SierraBibUpdatePoller-qa](https://us-east-1.console.aws.amazon.com/lambda/home?region=us-east-1#/functions/SierraBibUpdatePoller-qa?tab=configure)
+ - [SierraBibDeleteUpdatePoller-qa](https://us-east-1.console.aws.amazon.com/lambda/home?region=us-east-1#/functions/SierraBibDeleteUpdatePoller-qa?tab=configure)
+ - [SierraItemUpdatePoller-qa](https://us-east-1.console.aws.amazon.com/lambda/home?region=us-east-1#/functions/SierraItemUpdatePoller-qa?tab=configure)
+ - [SierraItemDeleteUpdatePoller-qa](https://us-east-1.console.aws.amazon.com/lambda/home?region=us-east-1#/functions/SierraItemUpdatePoller-qa?tab=configure)
+ - [SierraHoldingUpdatePoller-qa](https://us-east-1.console.aws.amazon.com/lambda/home?region=us-east-1#/functions/SierraHoldingUpdatePoller-qa?tab=configure)
+ - [SierraHoldingDeleteUpdatePoller-qa](https://us-east-1.console.aws.amazon.com/lambda/home?region=us-east-1#/functions/SierraHoldingDeleteUpdatePoller-qa?tab=configure)
+
+To enable/disable the trigger:
+ - On the Lambda page, follow Configuration > Trigger
+ - Click on the EventBridge rule
+ - Click Disable or Enable
+
+## Syncing Production Elasticsearch to QA
+
+The following describes how to copy production down to QA using logstash with no downtime.
+
+> Note that this method took 3 days to complete the first time it was run because it had to be resumed frequently due to dropped Internet. The second time it was run it ran continuously for 10 hours.
+
+**1. Enable access**
+
+Ensure you have access to both domains by [adding your IP to the Access Policy for each](https://github.com/NYPL/aws/blob/master/common/elasticsearch.md#2-make-the-domain-public-restrict-by-ip). (You should revert this later.)
+
+**2. Prepare the destination index**
+
+Prepare the index using `index-admin prepare` to apply mapping, etc.:
+
+```
+node jobs/index-admin prepare --index resources-[YYYY-MM-DD] --profile nypl-digital-dev --envfile config/qa.env
```

Comment:
where does jobs/index-admin come from?

Reply:
Added clarity. Will put this PR in Draft form until we can simplify this step.